packages/raystack/avatar/avatar.tsx

```diff
@@ -128,12 +157,12 @@ export const AvatarGroup = forwardRef<HTMLDivElement, AvatarGroupProps>(
           </div>
         ))}
         {count > 0 && (
-        <div className={styles.avatarWrapper}>
+          <div className={styles.avatarWrapper}>
             <Avatar
               size={avatars[0].props.size}
               radius={avatars[0].props.radius}
               variant={avatars[0].props.variant}
-              color='indigo'
+              color='neutral'
               fallback={<span>+{count}</span>}
```

Comment:
Can you add support for `className`, so the user can have options to customize it?

Reply:
added